src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1022,6 +1073,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             // macro directly with a VirtualKeyModifiers
             const auto shiftEnabled = WI_IsFlagSet(modifiers, static_cast<uint32_t>(VirtualKeyModifiers::Shift));
 
+            if (_TrySendMouseEvent(point, true))

```

Comment:
`false` should be passed in here for `goingDown` right?

Reply:
wow. This was a really messy PR haha. Thanks!